nx.json

```diff
@@ -11,7 +11,8 @@
           "build",
           "lint",
           "test",
-          "e2e"
+          "e2e",
+          "component-test"
         ],
         "parallel": 3,
         "accessToken": "MmZlZGE0MmUtODY0Zi00OGM5LTg0MGItNmFjMzUzMzE3OWY2fHJlYWQtd3JpdGU="
```

Comment:
Seems odd that nx would put this on a config that is not git ignored.

Reply:
According to [the docs](https://nx.dev/nx-cloud/account/access-tokens), removing the `accessToken` property effectively disables nx cloud. We are not actually using nx cloud, so I decided to remove this.